README.md

```diff
@@ -535,6 +535,41 @@ No additional changes are required to run the pipeline except updating the `--fi
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.parquet"
 ```
 
+#### 6. ORC
+
+##### Inspection of ORC files
+The inspection pipeline will read ORC files from input GCS storage bucket and the results of will be written in BigQuery
+tables.
+
+```commandline
+./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.orc"
+```
+
+##### De-identification
+The de-identification pipeline support ORC file format where data can be read from input GCS storage bucket and the 
+de-identified results can be written in 
+1. output BigQuery dataset as tables, or 
+
+```commandline
+./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.orc"
+```
+
+2. output GCS storage bucket as de-identified ORC files.
+
+```commandline
+./gradlew run ... -Pargs="... 
+--filePattern=gs://${PROJECT_ID}-demo-data/*.orc \
+--outputBucket=<output_storage_bucket> ..."
+```
+
+In the above command, replace <i>output_storage_bucket</i> with the GCS storage bucket URI where you want to store the 
+de-identified ORC files.
+
+Input files with varying schema is supported by the de-identification pipeline. Currently, it can process only Primitive
+data types available in ORC format when the results are stored in output GCS storage bucket.
+
+For sample data in ORC file format, refer [mock-data](.github/mock-data).
```

Comment:
refer --> refer to

There's another instance of this in the Parquet section. Please edit that too.